### ISSUE TITLE
Cut a new release

### ISSUE THREAD

**Issue by nullobject** [mendelt/embedded-midi#8]
There are some fixes in the `midi-convert` crate that fixed issues where the MSB & LSB were swapped when parsing 14-bit values:

* [Swap MSB and LSB in SongPositionPointer](https://github.com/rust-midi/midi-convert/commit/256da88abcdc21aa11ae9510f8689d54df2bc22e)
* [Swap pitch bend LSB and MSB](https://github.com/rust-midi/midi-convert/commit/817e55016a877e7f98b206bec88b4aca69dfbf8b)

Please cut a new release of `embedded-midi` so these fixes are available in this crate.

**Comment by everdrone** [comment]
Also looking forward to this. Did you find an alternative crate to handle midi?

**Comment by nullobject** [comment]
@everdrone No. I ended up installing this crate from a git ref, but it feels like this project has been abandoned which is a shame.

**Comment by mendelt** [comment]
Sorry, i havent had much time for this project lately and hadn't noticed there was demand for it. I'll roll a release when i have time over the holidays.

**Comment by laenzlinger** [comment]
would it make sense to upgrade to  embedded-hal = "1.0.0" ?

**Comment by laenzlinger** [comment]
I have created a PR for the upgrade to embedded hal 1.0.0 https://github.com/rust-midi/embedded-midi/pull/9

**Comment by mendelt** [comment]
Thank you, its merged. I should have a new release out somewhere next week. I hadnt looked at this code for over a year so its taking me some time to get back into it.
